tumblr_client.gemspec

```diff
@@ -4,14 +4,14 @@ require File.join(File.dirname(__FILE__), 'lib/tumblr/version')
 Gem::Specification.new do |gem|
   gem.add_dependency 'faraday', '~> 0.9.0'
   gem.add_dependency 'faraday_middleware', '~> 0.9'
-  gem.add_dependency 'json'
-  gem.add_dependency 'simple_oauth'
-  gem.add_dependency 'oauth'
-  gem.add_dependency 'mime-types'
-  gem.add_development_dependency 'rake'
-  gem.add_development_dependency 'rspec'
-  gem.add_development_dependency 'webmock'
-  gem.add_development_dependency 'simplecov'
+  gem.add_dependency 'json', '~> 1.8'
```

Comment:
The gemspec format does not support dynamic dependencies based on ruby versions, so you have to resort to an ugly but unavoidable workaround: conditionalize the dependencies using RUBY_VERSION like this: https://github.com/sparklemotion/http-cookie/blob/master/http-cookie.gemspec#L28-L34

One thing the maintainers need to make sure is to generate a gem for release with the latest ruby so the gem will have the most lax dependencies in its compiled spec.  Users of the gem must take care of version constraints on their own if they use old rubies, but that's inevitable.